src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.styles.ts

```diff
@@ -1,17 +1,29 @@
 import { CardboardClassNamePrefix } from '../../../../../Models/Constants/Constants';
-import { IConditionsListStyles } from './ConditionsList.types';
+import {
+    IConditionsListStyles,
+    IConditionsListStylesProps
+} from './ConditionsList.types';
 
 const classPrefix = CardboardClassNamePrefix + '-conditions-list';
 const classNames = {
     container: `${classPrefix}-container`
 };
 
-export const getStyles = (): IConditionsListStyles => {
+export const getStyles = (
+    props: IConditionsListStylesProps
+): IConditionsListStyles => {
     return {
         container: [
             classNames.container,
-            // 29px is label size, since this is resizing need to calc the 100% - label
-            { overflow: 'auto', height: 'calc(100% - 29px)' }
-        ]
+            // 30px is label size, since this is resizing need to calc the 100% - label
+            { overflow: 'auto', height: 'calc(100% - 30px)' }
+        ],
+        subComponentStyles: {
+            addButton: {
+                root: {
+                    color: props.theme.palette.themePrimary
```

Comment:
there's probably a semantic color for primary button color. Not a big deal but can use that in the future.